docs/deployments/migrate-overview.mdx

```diff
@@ -62,7 +62,7 @@ As part of the trickle migration, you'll need to determine an appropriate length
 
 #### Dealing with the delta
 
-A trickle migration is great for upgrading active users and sessions to take advantage of Clerk but there will always be some users who won't create an active session within the migration window and therefore will need to be migrated by other means, typically via [Basic Import / Export](#basic-export-import). However, the benefit of a trickle migration is that the risk of coordination becomes much lower as the delta users are likely to be less active, are less in numbers, and therefore the risk of data de-synchronization is reduced.
+A trickle migration is great for upgrading active users and sessions to take advantage of Clerk but there will always be some users who won't create an active session within the migration window and therefore will need to be migrated by other means, typically via [Basic Import / Export](#basic-export-import). A trickle migration lowers coordination risk because the remaining (in-active) users are fewer, so data de-synchronisation risk drops, too.
```

Comment:
nit
```suggestion
A trickle migration is great for upgrading active users and sessions to take advantage of Clerk, but there will always be some users who won't create an active session within the migration window and will need to be migrated by other means, typically via [Basic Import / Export](#basic-export-import). A trickle migration lowers coordination risk because the remaining (in-active) users are fewer, so data de-synchronisation risk drops, too.
```

Reply:
yeah, that's fair. I just pushed a commit to adjust and to use the US spelling of "synchronization".